playbooks/network-ee-integration-tests/pre.yaml

```diff
@@ -61,7 +61,7 @@
           appliance:
             vars:
               ansible_python_interpreter: /bin/python3.8
-      when: container_image_version in ["stable-2.9", "stable-2.11"]
+      when: container_image_version in ["stable-2.9", "stable-2.11"] and ansible_connection != "ssh"
```

Comment:
this won't work, as ansible_connection is set from the controller vars.

What we need to do likey, is inspect the _inventory variable from line 32 slurp. And check if the inventory file 'appliance' vars is set to it

Reply:
sorry, _inventory_data on line 39

Reply:
okay, what we should maybe do is delete this logic and create a new job variable called:

test_ansible_python_interpreter

that when set, we set the value.  Like we do for test_ansible_ssh_private_key_file

This will then mean, all existing stable-2.11 / stable-2.9 network-ee integration jobs need to be updated to set test_ansible_python_interpreter: /usr/bin/python3.8

But even that means we need to update a bunch of things....

Reply:
let me think more about it tonight

Reply:
Yah, lets go the test_ansible_python_interpreter route. then stable jobs can set it